sites/partners/src/pages/listings/[id]/applications/index.tsx

```diff
@@ -160,86 +170,122 @@ const ApplicationsList = () => {
 
       <ListingStatusBar status={listingDto?.status} />
 
-      <section className={"bg-gray-200 pt-4"}>
-        <article className="flex flex-col md:flex-row items-start gap-x-8 relative max-w-screen-xl mx-auto pb-8 px-4 flex-col">
-          {listingDto && (
-            <>
-              <ApplicationsSideNav
-                className="w-full md:w-72"
-                listingId={listingId}
-                listingOpen={isListingOpen}
-              />
-
-              <AgTable
-                className="w-full"
-                id="applications-table"
-                pagination={{
-                  perPage: tableOptions.pagination.itemsPerPage,
-                  setPerPage: tableOptions.pagination.setItemsPerPage,
-                  currentPage: tableOptions.pagination.currentPage,
-                  setCurrentPage: tableOptions.pagination.setCurrentPage,
-                }}
-                config={{
-                  gridComponents,
-                  columns: columnDefs,
-                  totalItemsLabel: t("applications.totalApplications"),
-                }}
-                data={{
-                  items: applications,
-                  loading: appsLoading,
-                  totalItems: appsMeta?.totalItems,
-                  totalPages: appsMeta?.totalPages,
-                }}
-                search={{
-                  setSearch: tableOptions.filter.setFilterValue,
-                }}
-                sort={{
-                  setSort: tableOptions.sort.setSortOptions,
-                }}
-                headerContent={
-                  <div className="flex-row">
-                    {allowNewApps && (
-                      <Button
-                        href={`/listings/${listingId}/applications/add`}
-                        variant="primary-outlined"
-                        size="sm"
-                        className="mx-1"
-                        id={"addApplicationButton"}
-                      >
-                        {t("applications.addApplication")}
-                      </Button>
-                    )}
-
-                    <Button
-                      variant="primary-outlined"
-                      size="sm"
-                      className="mx-1"
-                      onClick={() => setIsTermsOpen(true)}
-                      loadingMessage={csvExportLoading && t("t.formSubmitted")}
+      {shouldExpireData && expiryDate <= dayjs() ? (
+        <section className={pageStyles["expired"]}>
+          <div className={pageStyles["parent"]}>
+            <div className={pageStyles["container"]}>
+              <div className={pageStyles["main"]}>
+                <Card spacing={"lg"}>
+                  <CardSection>
+                    <Icon icon={faExclamationCircle} size="xl" />
```

Comment:
Should this be hero icon?

Reply:
The Hero icons haven't made their way into Doorway from core yet, and there's a Hero custom icon mapper added in that PR we should use